cou/apps/auxiliary.py

```diff
@@ -93,3 +94,29 @@ def channel_codename(self) -> OpenStackRelease:
                 f"for channel: '{self.channel}'"
             )
         )
+
+
+@AppFactory.register_application(["rabbitmq-server"])
+class RabbitMQServer(OpenStackAuxiliaryApplication):
+    """RabbitMQ application.
+
+    RabbitMQ had to wait until the entire model will be idle after the upgrade to ensure other
+    upgrade can continue.
+    """
+
+    def post_upgrade_plan(self, target: OpenStackRelease) -> list[Optional[UpgradeStep]]:
+        """Adding wait to post upgrade step.
+
+        :param target: OpenStack release as target to upgrade.
+        :type target: OpenStackRelease
+        :return: Plan that will add pre upgrade as sub steps.
+        :rtype: list[Optional[UpgradeStep]]
+        """
+        wait = UpgradeStep(
+            description=f"Check if the workload of '{self.name}' has been upgraded",
```

Comment:
this needs to be updated, it's the same output of `_get_reached_expected_target_plan()`

Reply:
Yes, I copy pasted it and forgot to change it. Thanks